lib/order_optimizer.rb

```diff
@@ -55,12 +55,18 @@ def find_possible_orders(required_qty:, skus:)
   def count_and_remainder_for_sku(quantity, sku)
     count, remainder = quantity.divmod(sku.quantity)
 
-    if sku.min_quantity && count * sku.quantity < sku.min_quantity
+    if sku.max_quantity && count * sku.quantity > sku.max_quantity
```

Comment:
I wonder if this lends itself to a range, something like `if sku.quantities.cover?(count * sku.quantity)` where `sku.quantities` is defined as `(min_quantity..max_quantity)`? Could be a nice refactor, as an idea.